mathics/builtin/box/graphics.py

```diff
@@ -1066,7 +1070,9 @@ def init(
             if len(item.elements) not in (1, 2, 3):
                 raise BoxConstructError
             content = item.elements[0]
-            self.content = content.format(graphics.evaluation, "TraditionalForm")
+            self.content = format_element(
+                content, graphics.evaluation, "TraditionalForm"
```

Comment:
One thing I noticed in general looking things over is that "format_element" or ".format", really should be taking a Symbol parameter as we did for Expression.  If I recall correctly, when I looked at the code, the first thing it does is call Symbol on the `str` parameter. 

This can come later though. But I thought I would point it out.